sycl/test-e2e/format.py

```diff
@@ -98,6 +98,47 @@ def getMatchedFromList(self, features, alist):
         except ValueError as e:
             raise ValueError("Error in UNSUPPORTED list:\n%s" % str(e))
 
+    def make_default_features_list(self, expr, triple, add_default=True):
+        ### EXCEPTIONS LIST
+        # TODO: Define elsewhere?
+        exceptions = {}
+        exceptions["spir64"]={
+                "cuda":False, "hip":False, "hip_amd":False, "hip_nvidia":False,
+                "native_cpu":False,
+                }
+        exceptions["system"]={
+                "linux":True, "windows":False, "system-windows":False,
+                "run-mode":False, "TEMPORARY_DISABLED":False,
+                }
+        queried_features = []
+        for f in expr:
+            queried_features = queried_features + re.findall("[-+=._a-zA-Z0-9]+", f)
```

Comment:
what's a queried feature?

Reply:
a feature that exists within the lit line. i.e., if we have `REQUIRES: cpu, sg-32` the queried features for this line would be the list `[cpu, sg-32]`